src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.star-container {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 12px;
+  width: 12px;
+  margin: 2px 4px;
+  background-size: cover;
+}
+
+.stars__star:first-child {
+  margin-left: 2px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(n) {
```

Comment:
again, if I have 7 stars per row, I use this modifier (stars--5), then I expect only 5 stars to have a different background. in your case all 7 stars will have different background, not 5 as this modifier says